larva/src/main/java/org/frankframework/larva/LarvaTool.java

```diff
@@ -888,7 +889,19 @@ public String initScenariosRootDirectories(String realPath, String paramScenario
 	public List<File> readScenarioFiles(AppConstants appConstants, String scenariosDirectory) {
 		List<File> scenarioFiles = new ArrayList<>();
 		debugMessage("List all files in directory '" + scenariosDirectory + "'");
-		File[] files = new File(scenariosDirectory).listFiles();
+
+		File directory = new File(scenariosDirectory);
+		Path targetPath = directory.toPath().normalize();
+
+		if (!directory.toPath().normalize().startsWith(targetPath)) {
+			String message = "Scenarios directory is outside of the target directory";
+			logger.warn(message);
+			writeLog(XmlEncodingUtils.encodeChars(message) + "<br/>", LarvaLogLevel.ERROR, false);
```

Comment:
Niels, zoiets bedoel je denk ik?

Reply:
Ja al snap ik niet precies waar de `XmlEncodingUtils.encodeChars` voor nodig is